test/unit/lib/plugins/aws/package/compile/events/schedule.test.js

```diff
@@ -102,15 +103,38 @@ describe('test/unit/lib/plugins/aws/package/compile/events/schedule.test.js', ()
           },
         },
       },
+      {
```

Comment:
New tests should be configured purely with `runServerless` util. See: https://github.com/serverless/serverless/tree/main/test#unit-tests

In existing file we usually then introduce a new describe block for those tests (and old ones ideally at some point should be refactored to also go there), e.g. check https://github.com/serverless/serverless/blob/741847d2327e87eb95b9a2349642fac5b7e316e1/test/unit/lib/plugins/aws/package/compile/events/cloud-front.test.js#L171

Reply:
@medikoo Does this line in `schedule.test.js` not setup the whole file to run using `runServerless`? https://github.com/serverless/serverless/blob/bf79934e9b43a0b2e8613ee29f9430fa22c41481/test/unit/lib/plugins/aws/package/compile/events/schedule.test.js#L23

Reply:
@joelwalden indeed! Sorry I got fooled by the not common `runServerless` setup. Ignore my comment then